### PR TITLE
Adjust the script to handle next and latest published 

### PR DESCRIPTION
**Pull request by lmcbout**
Adjust package.json to use the script
Add script to use the next | latest for Theia dependencies
Prevent git to auto-commit the code when publishing

Signed-off-by: Jacques Bouthillier <jacques.bouthllier@ericsson.com>

### CODE REVIEW

**Comment by vince-fugnitto** [comment]
@lmcbout please check the failed eca check :(

File: scripts/theia-version.sh [hunk 1]
```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
```
Comment (vince-fugnitto) [inline scripts/theia-version.sh]: @lmcbout was the code copied from anywhere?

File: lerna.json [hunk 2]
```diff
@@ -1,11 +1,14 @@
 {
   "lerna": "2.4.0",
-  "version": "0.0.0",
+  "version": "0.7.0",
```
Comment (vince-fugnitto) [inline lerna.json]: Based on discussions with Marc, we should keep the version as `0.0.0`
(and align the `cpp-debug` and `cortex-debug` to `0.0.0`)
This means that when we publish (the first latest), `lerna` will step the versions properly at `0.1.0`.

cc @marcdumais-work

**Comment by lmcbout** [comment]
Wrong sign-off committer, will re-push it

**Comment by lmcbout** [comment]
Push on a different branch

**Pull request closed without merging**